library/item/item_setting/src/main/java/org/quiltmc/qsl/item/setting/api/RecipeRemainderLogicHandler.java

```diff
@@ -50,12 +50,15 @@ public interface RecipeRemainderLogicHandler {
 	 * @return the recipe remainder
 	 */
 	static ItemStack getRemainder(ItemStack original, @Nullable Recipe<?> recipe, RecipeRemainderLocation location) {
-		Map<RecipeRemainderLocation, RecipeRemainderProvider> providers = CustomItemSettingImpl.RECIPE_REMAINDER_PROVIDER
-				.get(original.getItem());
+		Map<RecipeRemainderLocation, RecipeRemainderProvider> providers =
+			CustomItemSettingImpl.RECIPE_REMAINDER_PROVIDER.get(original.getItem());
 
-		RecipeRemainderProvider provider = (_original, _recipe) -> _original.getItem().hasRecipeRemainder() ? _original.getItem().getRecipeRemainder().getDefaultStack() : ItemStack.EMPTY;
+		RecipeRemainderProvider provider = (_original, _recipe) -> _original.getItem().getRecipeRemainder();
```

Comment:
Is there a reason this was modified? I'm fairly certain I added this to fix a bug

Reply:
`hasRecipeRemainder` doesn't exist anymore, but `getRecipeRemainder` is `final` and returns `ItemStack.EMTPY` if the remainder is `null`